test/unit/rules/lint-attribute-indentation-test.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'attribute-indentation',
+
+  config: true,
+
+  good: [
+    //Non Block form
+    '{{contact-details' + '\n' +
+    '  firstName=firstName' + '\n' +
+    '  lastName=lastName' + '\n' +
+    '}}',
+    //Non Block form with no params
+    '{{contact-details}}',
+    //Block form
+    '{{#contact-details' + '\n' +
+    '  firstName=firstName' + '\n' +
+    '  lastName=lastName' + '\n' +
+    'as |fullName|}}' + '\n' +
+    '  {{fullName}}' + '\n' +
+    '{{/contact-details}}',
+    //Block form with no params
+    '{{#contact-details' + '\n' +
+    'as |contact|}}' + '\n' +
+    '  {{contact.fullName}}' + '\n' +
+    '{{/contact-details}}'
+  ],
+
+  bad: [{
+      //MustacheStatement
+    template: '{{contact-details firstName=firstName lastName=lastName}}',
```

Comment:
I think the rule needs to be a bit nuanced here.  I think this should be a "good" example (non-block single line invocation with less than 80 chars).